google-cloud-compute-v1/.repo-metadata.json

```diff
@@ -5,5 +5,6 @@
     "name_pretty": "Google Cloud Compute V1 API",
     "release_level": "alpha",
     "repo": "googleapis/google-cloud-ruby",
-    "requires_billing": true
+    "requires_billing": true,
+    "library_type": "GAPIC_MANUAL"
```

Comment:
Hold off on this one for now. It's actually generated; Slava is still working on getting it into synth/owlbot.